nvfuser/__init__.py

```diff
@@ -10,6 +10,14 @@
 # This is needed when libnvfuser.so is patched and doesn't have the pytorch library location available.
 sys.path.append(os.path.join(os.path.dirname(torch.__file__), "lib"))
 
+cmake_prefix_path = os.path.join(
```

Comment:
Can we do this inside `./nvfuser_python_utils/__init__.py`? We do need to adjust the cmake path there, but that's a pretty lightweight import at least. :laughing:

Reply:
I created `nvfuser/utils.py` since it seemed more intuitive with the submodule build.